lib/unifile-dropbox.js

```diff
@@ -51,10 +51,13 @@ function callAPI(session, path, data, subdomain = 'api', isJson = true, headers
 			}
 			if(res.statusCode >= 400) {
 				return reject({statusCode: res.statusCode, message: (() => {
-					console.error('Error in unifile', body.toString());
+					const bodyStr = body.toString();
```

Comment:
Could you test if body is already an object and if it has a property named `error_summary`. If it does, return it, if not return as-is

Reply:
Yes ok
I will remove the try/catch then

Reply:
as we discussed the try catch is needed 
should I merge?

Reply:
Actually the body is `JSON` when `isJson === true` . So we should test its value.